src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -592,7 +592,7 @@ bool ViewProviderSketch::mouseButtonPressed(int Button, bool pressed, const SbVe
                                             const Gui::View3DInventorViewer *viewer)
 {
     assert(edit);
-    App::AutoTransaction comitter;
+    App::AutoTransaction committer;

```

Comment:
This is a variable name, all other occurrences should be changed too

Reply:
@yorikvanhavre hmm..yea, I believe I did check that at the time..and I just double checked and that is the only occurrence of that variable name in the code.

Reply:
It must certainly be used somewhere (otherwise the compiler give warnings)... I'll have a better look when i have a moment

Reply:
That's OK. There is no need to call any of the methods of AutoTransaction because all the magic happens inside the constructor and destructor. There might a certain warning switch where the compiler will complain about but in this case the compiler is wrong.

A similar thing is the class PyGILStateLocker whose only purpose is to lock and release the GIL. The class only declares a constructor and destructor.

Both classes use the RAII idiom of C++:
https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization

Reply:
Thanks for the cool explanation :) Interesting thing... OK then!

Reply:
So i should whitelist `comitter` here when running typo script ?